pkg/machine/ignition/ignition.go

```diff
@@ -308,6 +309,21 @@ ExecStart=-/usr/sbin/agetty --autologin root --noclear %I $TERM
 		ignSystemd.Units = append(ignSystemd.Units, recoveryUnit)
 	}
 
+	// Only AppleHv has the rosetta setting
```

Comment:
all these changes should go to: https://github.com/containers/podman/blob/bed619ca3d981082a334c4939825ac9d96de4065/pkg/machine/applehv/stubber.go#L338

Reply:
@baude Fixed in https://github.com/containers/podman/pull/21670/commits/f0896648a82375bf0548de8801faa9138dda8207
Please check.